autolens/pipeline/phase/dataset/analysis.py

```diff
@@ -72,11 +72,13 @@ def associate_hyper_images(self, instance: af.ModelInstance) -> af.ModelInstance
            The input instance with images associated with galaxies where possible.
         """
         if self.hyper_galaxy_image_path_dict is not None:
```

Comment:
Could use try - except rather than checking if things are None here

Reply:
Skipping for now - the None is used for visualization too so will address this in future.